### PR TITLE
Feature/membersbrowser 0.3.0 20160705

### PR DESCRIPTION
**Pull request by soyerr**
For test

### CODE REVIEW

File: distributes/jbb-web-app/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <parent>
         <artifactId>distritubes</artifactId>
         <groupId>org.jbb.dist</groupId>
-        <version>DEV-SNAPSHOT</version>
+        <version>0.3.0-membersbrowser-SNAPSHOT</version>
```
Comment (soyerr) [inline distributes/jbb-web-app/pom.xml]: Test comment

Comment (soyerr) [inline distributes/jbb-web-app/pom.xml]: !completed

Comment (soyerr) [inline distributes/jbb-web-app/pom.xml]: !fix

File: implementation/domain/api/jbb-members-api/src/main/java/org/jbb/members/api/services/MemberService.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/*
```
Comment (soyerr) [inline implementation/domain/api/jbb-members-api/src/main/java/org/jbb/members/api/services/MemberService.java]: Pro !resolved

File: implementation/domain/logic/jbb-frontend-web/src/main/java/org/jbb/frontend/web/controllers/HomePageController.java [hunk 3]
```diff
@@ -42,12 +42,6 @@ public String faq() {
         return "faq";
     }
 
-    @RequestMapping("/members")
```
Comment (soyerr) [inline implementation/domain/logic/jbb-frontend-web/src/main/java/org/jbb/frontend/web/controllers/HomePageController.java]: Moved to other class !completed

File: distributes/jbb-web-app/pom.xml (hunk 1 above)
Comment (baart92) [inline distributes/jbb-web-app/pom.xml]: Comment test !fix

**Pull request merged**